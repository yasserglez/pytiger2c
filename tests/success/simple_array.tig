let
    type arr = array of string
    type arr2 = array of int
    var a := arr[10] of "output\n"
    var b := arr2[10] of 0
in
    print(a[0]);
    a[1] := "new output\n";
    print(a[1]);
    b[1] := 2;
    printi( b[0]);
    print("\n");
    printi( b[1]);
    print("\n")
end
