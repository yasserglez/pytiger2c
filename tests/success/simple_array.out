output
new output
0
2
